src/form-header.scss

```diff
@@ -0,0 +1,31 @@
+@import './new-settings';
+@import './mixins';
+
+/*!
+.fd-form-header
+   .fd-form-header__text
+*/
+$block: #{$fd-namespace}-form-header;
+$fd-form-header-padding-x: 1rem !default;
+
+.#{$block} {
+  @include fd-reset();
+  @include fd-flex-vertical-center();
+
+  width: 100%;
+  height: 2.75rem;
+  background-color: var(--sapGroup_TitleBackground);
+  padding-left: $fd-form-header-padding-x;
+  padding-right: $fd-form-header-padding-x;
+  box-shadow: 0 var(--sapList_BorderWidth) var(--sapList_BorderWidth) var(--sapGroup_TitleBorderColor);
```

Comment:
![image](https://user-images.githubusercontent.com/26483208/97993639-a13fe400-1de4-11eb-8a24-04091b19958c.png)
Can you add new parameter to theming, so for high contrast there will be `3 * --sapList_BorderWidth` and for rest `--sapList_BorderWidth`?